example/lib/main.dart

```diff
@@ -111,6 +111,14 @@ const _colorizeTextStyle = TextStyle(
   fontFamily: 'Horizon',
 );
 
+const _flickerTextStyle = TextStyle(fontSize: 35, shadows: [
+  Shadow(
+    blurRadius: 7.0,
+    color: Colors.white,
+    offset: Offset(0, 0),
+  ),
+]);
+
```

Comment:
This is the reason for the glowing effect, maybe we can inherently make this a part of this Animated Text? Any suggestions @awhitford @SirusCodes

Reply:
Including it in the package will make it difficult for users to edit its behaviour 🤔 If there is a simple way to do it then it would be really great.

Reply:
Makes sense!